test/test_torch.py

```diff
@@ -3772,6 +3772,31 @@ def get_probs(length, is_contiguous):
                 a_t, p_t = torch._multinomial_alias_setup(probs)
                 torch._multinomial_alias_draw(p_t.view(2, 2), a_t.view(2, 2))
 
+        for probs in [get_probs(4, True), torch.tensor([0.8, 0.2]), torch.tensor([0.7, 0.2, 0.1])]:
+            # Check how different the alias distribution and the original distribution are
+            n_samples = 50000
+            alias_table, prob_table = torch._multinomial_alias_setup(probs)
+            alias_samples = torch._multinomial_alias_draw(prob_table, alias_table, n_samples)
+            alias_dist = torch.unique(alias_samples, return_counts=True)[1].to(dtype=probs.dtype) / n_samples
+            rtol = 0.1 if IS_WINDOWS else 0.02
```

Comment:
Is there a known reason why the tolerance needs to be higher for Windows?

Reply:
I don’t know, the tests don’t seem to pass otherwise.

Reply:
Windows tests don’t pass with 0.1, I guess I’ll have to print out the values to see what exactly is going on.

Reply:
I see. I am fine merging this PR since this is independent of the changes that you have made, but it may be worth understanding why Windows requires a lower precision.